cppsrc/U8x8lib.cpp

```diff
@@ -814,6 +823,14 @@ extern "C" uint8_t u8x8_byte_arduino_hw_spi(u8x8_t *u8x8, uint8_t msg, uint8_t a
 extern "C" uint8_t u8x8_byte_arduino_2nd_hw_spi(U8X8_UNUSED u8x8_t *u8x8, U8X8_UNUSED uint8_t msg, U8X8_UNUSED uint8_t arg_int, U8X8_UNUSED void *arg_ptr)
 {
 #ifdef U8X8_HAVE_2ND_HW_SPI
+  /* ISSUE #991 */
+  #ifdef SPI_RESET_FUNC 
```

Comment:
Why SPI_RESET_FUNC? The #ifdefs should match, so this should test for STM32...

Reply:
I'm opening an issue to the SPI lib. In Arduino_STM32 library. I don't know why but something is broken there and not in your lib. In fact i'm doing only few modifications only to get sure that all works. Maybe we can check for the SPI _currentSettings if is NULL. 
but the spi_dev inside Is private. And no way to setup and this is required to have SPI communications.

Reply:
ok...